src/main.ts

```diff
@@ -95,6 +95,35 @@ export default class DeepLPlugin extends Plugin {
 			},
 		});
 
+		this.addCommand({
+            id: "translate-and-append-selection-automatically",
+            name: "Translate selection: Automatically append selection",
+            editorCallback: async (editor: Editor) => {
+                const selection = editor.getSelection();
+                if (selection === "") {
+                    new Notice("No text selected.");
+                    return;
+                }
+
+                try {
+                    const translation = await this.deeplService.translate(
+                        selection,
+                        this.settings.toLanguage,
+                        this.settings.fromLanguage
+                    );
+                    const textToAppend = this.settings.appendNewLine ? '\n' + translation[0].text : translation[0].text;
```

Comment:
I think it would be good if the setting for the new line is respected in all commands that keep the original text. 

For example: [deepl-translate-selection-append](https://github.com/friebetill/obsidian-deepl/pull/12/files#diff-4fab5baaca5c14d2de62d8d2fceef376ddddcc8e9509d86cfa5643f51b89ce3dR81)

Reply:
I only changed it for the setting(s) which appended. I didn't want to change existing functions.

Reply:
I can understand that. But it would make sense to me if all functions that append would also respect the setting of the new line. With your change, not all would be the same. If the default for the setting is false, it does not change anything for existing functions, unless you set it to true. From a user's perspective, it would irritate me if one command made a new line and another did not. I would therefore adapt it everywhere for the sake of consistency.

But that's just my opinion, I'm not the owner. Not meant to sound offensive, sorry if it was :)

Reply:
No no! I am new to this, and I'm not immune from mistakes or criticisms. Also I appreciate you taking the time to reply!
I'm of course a bit excited since this is really my first pull request, and I shouldn't expect it to be perfect or even accepted. 

Regarding the new line, you are right. I was focused on the append parts, and I didn't take too much care of the other functions either. I just translate a lot of things, and I didn't want the existing stuff to be overwritten, and I needed a fast way to compare translations as well. Thus this 'stacked' method and a focus on the appending.

Reply:
It takes a lot of conviction to change someone else's code, so a certain amount of conviction is necessary. The same with commenting and reviewing. We are all not free from errors. As long as we can talk about it, it's good, we learn something new every day. 

And I can understand that from your perspective. I tried to take a different perspective. If we're of the same opinion now, I'd say the discussion was worthwhile :)